modules/ROOT/partials/configuration/tinycomments_delete.adoc

```diff
@@ -8,7 +9,7 @@ The `tinycomments_delete` function is passed a (`req`) object as the first param
 
 The `done` callback should accept the following object:
 
-[source]
+[source, ts]
```

Comment:
The reason I used ts here is to improve the syntax highlighting because this specific code block does not contain code but interface definition. Should I still revert this change or should I keep it ?

Reply:
I more meant that the others should be converted to `ts` as you've used it elsewhere below as well and I agree `ts` is the better option.

Reply:
Done in tinycomments_create, tinycomments_reply and tinycomments_resolve